conf_open_source.py

```diff
@@ -53,17 +53,26 @@
     #{'name': 'flow_test'},
 ]
 
+DEVTOOLSET = [
```

Comment:
Just declare it as:
DEVTOOLSET = 'scl enable devtoolset-6 bash'
OR
DEVTOOLSET = 'source /opt/rh/devtoolset-6/enable'

and run it as:
cmd=f'{DEVTOOLSET} && perl tools/builder/build_mfx.pl --cmake={CMAKE_CFG}'

Reply:
Ok, but I will fix it when I will write new configuration in closed source :)  
To have configuration in the same style.